blocks/library/embed/index.js

```diff
@@ -85,8 +91,14 @@ registerBlockType( 'core/embed', {
 			this.noPreview = [
 				'facebook.com',
```

Comment:
Eventually we'll move the embedded previews in the editor into sandbox iframes. Facebook embeds refused to run inside these iframes, and throw up errors, so we want to display a "sorry, preview not available" message instead.

Though, yeah, noPreview should be top level and named better. Will fix!